install_protoc_from_source.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env sh
+
+PROTOC_VERSION="2.6.1"
+
+if [[ ! -d "$HOME/protobuf/lib" ||  "$(protoc --version)" != *${PROTOC_VERSION}* ]]; then
```

Comment:
On the point of not being a standard install path, that's not really a good thing. Better would be to put it in `/opt` or `/usr/local` or ... Including the version in the path would address the version conflicts, akin to what Homebrew does.